src/codegen/compile.ml

```diff
@@ -1996,18 +1999,22 @@ module Tagged = struct
      so update all!
    *)
 
+  type bits_sort = U | S | F
```

Comment:
(Just cosmetics: Maybe a brief comment could be helpful saying what the abbreviations stand for.)

Reply:
Done